lib/Runtime/Library/JavascriptSimdInt8x16.cpp

```diff
@@ -6,7 +6,9 @@
 
 namespace Js
 {
-    JavascriptSIMDInt8x16::JavascriptSIMDInt8x16(SIMDValue *val, StaticType *type) : JavascriptSIMDType(type), value(*val)
+    const char16 JavascriptSIMDInt8x16::_typeName[] = _u("SIMD.Int8x16");
+
+    JavascriptSIMDInt8x16::JavascriptSIMDInt8x16(SIMDValue *val, StaticType *type) : JavascriptSIMDType(val, type)
```

Comment:
Why doesn't this have just the type constructor JavascriptSIMDFloat32x4(StaticType *type); ?